input/projects/why-join-wip.md

```diff
@@ -0,0 +1,51 @@
+
+.NET Foundation services aim to increase project stability and maturity and don't replace developers on member projects. Day-to-day development remains in the hands of project maintainers and contributors.
+
+Project benefits include:
+
+## Project guidance and mentoring
+
+New projects joining us will get mentorship on how to best run an open source community along with access to developers currently seeking .NET projects. We work with project leads to help ensure the project grows into a vibrant and welcoming community. And, of course, we love to share best practices for managing a friendly, collaborative, and innovative open source workplace, wherever you may be.
+
+## IP and legal
+
+Legal can be tricky. We can help. Projects in our organization will get assistance to organize and manage their IP assignments as well as infrastructure for tracking common, simple, Contribution License Agreements across all .NET Foundation projects. We can also help ensure that your collaborators respect the open source licenses and trademarks of projects assigned to the .NET Foundation. Need advice on legal issues? We offer support there as well.
+
+## Marketing and communications
+
+The more people who know about .NET, the better for the ecosystem. We work with Amazon, JetBrains, Microsoft, Progress, and other industry leaders to improve your project's exposure within the community at large. Your success is our success - we'd love to see you achieve the accolades you deserve!
+
+## CLA management
+
+A Contributor License Agreement (CLA) ensures a project has the necessary rights to code contributions.
+
+Our automated CLA system has been working well for GitHub-based projects, with a very high rate of developers able to easily sign their Contribution License Agreements electronically. This service is offered with full automated notification to the project committers regarding the CLA status of a pull request author.
+
+## Secret management
+
+Get a secure vault to store secrets related to your project (social media credentials, website logins, etc.). Share credentials securely between trusted maintainers on your project, and enjoy the peace of mind that they're safely saved
+
+## Code signing
+
+Code signing protects against content tampering. A signature guarantees the assets haven't been modified since the asset was signed.
+
+We've implemented an automated service to Authenticode signing your release MSIs, EXEs, and NuGet packages with the .NET Foundation code signing certificate. The certificate is well known to Windows Smart Screen so that people can install your open-source application without warnings. It also allows for consuming applications knowing that the binary downloaded is the one created on your build server.
+
+## Hosting
+
+We provide hosting for a number of .NET Foundation project resources in Microsoft Azure. We have a .NET Foundation Office 365 account with SharePoint Online, Exchange, and more.
```

Comment:
**QUESTION:** How to get this benefit?

Reply:
I don't know. However, that sort of documentation belongs somewhere else. If you think there aren't good docs on using benefits then you could create a new issue?